content/en/docs/release_notes/zigong.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Zigong release notes
+linkTitle: Zigong
+description: 5 Novermber 2021
+date: 2021-11-05
+Hide_readingtime: true
+---
+## Summary
+
+Added support for creating and editing flows in the UI with the new alpha [OAS flow-trigger](/docs/developer_guide/flows/flow_triggers/oas_flow_trigger) and added [documentation](/docs/developer_guide/flows/flow_triggers/oas_flow_trigger) with information on how to start using it and our roadmap for its development.
+
+{{% releasenotes/upgrade %}}
+
+<!-- ## Breaking changes -->
+
+## Features
+* #7104: Added the ability to create and edit flows with the alpha release of the [OAS flow-trigger](https://www.npmjs.com/package/@axway/api-builder-plugin-ft-oas). Now, flows can be created from the UI and invoked. :warning: Note that this module is currently WIP and should not be considered production ready.
+
+## Fixes
+* #7105: Response `body` is no longer logged at DEBUG to improve performance.
+* #7077: Fixed an issue in [arrow-orm](https://www.npmjs.com/package/arrow-orm) that, when composite Models had a [Date](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date) field was aliased (i.e. renamed), the model could not be used to create records, resulting in an error `"required field value missing"`. Now, records are created without error.
```

Comment:
arrow-orm isn't a public module so won't mean much to users, to them this is really an updated runtime thing